test/testpy/__init__.py

```diff
@@ -73,7 +73,9 @@ def GetCommand(self):
       # failure so such tests are also skipped.
       if ('--inspect' in flag[0] or \
           '--use-bundled-ca' in flag[0] or \
-          '--use-openssl-ca' in flag[0]) and \
+          '--use-openssl-ca' in flag[0] or \
+          '--tls-v1.0' in flag[0] or \
+          '--tls-v1.1' in flag[0]) and \
```

Comment:
There's a (pre-existing) bug here: only the first flag is checked.

Something like `// Flags: --predictable --tls-v1.0` would be ignored because it only looks at the `--predictable` flag. The logic should be this:
```python
flags = flags_match.group(1).strip().split()
if ('--inspect' in flags' or '--use-bundled-ca' in flags or ...):
  # ...
```
(The backslashes aren't necessary in a `(...)` expression, by the way.)

Reply:
Ah yes, I'll update shortly fixing this. Thanks!